### PR TITLE
migrate and update image_proc docs

### PR DESCRIPTION
**Pull request by mikeferguson**
* move component documentation from ros wiki, update for various changes
 * add tutorial on how to run components
 * update tutorial on debayer/rectify to use launch file
 * remove image_proc node, it has always been completely broken and the launch file has the same (but working) functionality
 * update launch file to support namespace parameter for tutorial

### CODE REVIEW

**Comment by mikeferguson** [comment]
Testing done:
 * builds locally with rosdoc2
 * launch file tutorial is fully tested
image

File: image_proc/doc/conf.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# Copyright 2019 Open Source Robotics Foundation, Inc.
```
Comment (ahcorde) [inline image_proc/doc/conf.py]:
```suggestion
# Copyright 2024 Open Source Robotics Foundation, Inc.
```

File: image_proc/doc/tutorials.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        )
+
+        return LaunchDescription([container])
+
+Using image_proc Launch File
+----------------------------
+Make sure your camera driver is running. To see the available raw
+image topics from compatible drivers you can check:
+
+```
```
Comment (ahcorde) [inline image_proc/doc/tutorials.rst]:
```suggestion
```bash
```

File: image_proc/doc/tutorials.rst [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+ros2 topic list | grep image_raw
```
Comment (ahcorde) [inline image_proc/doc/tutorials.rst]:
```suggestion
$ ros2 topic list | grep image_raw
```

File: image_proc/doc/tutorials.rst [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+Normally the raw image from the camera driver is not what you want
+for visual processing, but rather an undistorted and (if necessary)
+debayered image. This is the job of ``image_proc``. If you are
+running on a robot, it's probably best to run ``image_proc`` there.
+For example, if the driver is publishing topics ``/my_camera/image_raw``
+and ``/my_camera/camera_info`` you would do:
+
+```
```
Comment (ahcorde) [inline image_proc/doc/tutorials.rst]:
```suggestion
```bash
```

File: image_proc/doc/tutorials.rst [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Notice that we push our ``image_proc`` launch file down into the
+``/my_camera`` namespace, in which it subscribes to the ``image_raw``
+and ``camera_info`` topics. All output topics are likewise published
+within the ``/my_camera`` namespace.
+
+In a separate terminal (on your home machine, if you are running on a robot):
+
+```
```
Comment (ahcorde) [inline image_proc/doc/tutorials.rst]:
```suggestion
```bash
```

**Review by ahcorde — CHANGES_REQUESTED** [review]
same small comments and maybe add in the `README.md` the command to build the documentation ?

**Comment by mikeferguson** [comment]
All great suggestions - but I'm going to skip adding build instructions to the README since we aren't doing anything fancy - just using rosdoc2 in the standard way

**Pull request merged**